ELLocationTests/ELLocationTests.swift

```diff
@@ -23,77 +117,461 @@ class ELLocationTests: XCTestCase {
         super.tearDown()
     }
     
-    private func deliverLocationUpdate(subject: LocationManager, latitude: CLLocationDegrees, longitude: CLLocationDegrees) {
-        subject.locationManager(CLLocationManager(), didUpdateLocations: [CLLocation(latitude: latitude, longitude: longitude)])
+    func testCalculateAndUpdateAccuracyCrash() {
+        LocationAuthorizationService().requestAuthorization(.WhenInUse)
+        LocationAuthorizationService().requestAuthorization(.Always)
+        
+        let subject = LocationManager()
+
+        // no crash here is a test success
+        subject.locationManager(CLLocationManager(), didUpdateLocations: [CLLocation(latitude: 42, longitude: 42)])
```

Comment:
Honestly, I don't even know what this test does. [It was already there](https://github.com/Electrode-iOS/ELLocation/blob/master/ELLocationTests/ELLocationTests.swift#L46-L52), I just modified it to not use the `shared` instance. Since the internal code has changed anyway, should we just remove this test?